ScreeningEncounterForm.html

```diff
@@ -0,0 +1,1209 @@
+	{"label": "Elfas  Limo", "val": "3648-3"},
+	{"label": "Wilson letting", "val": "3649-1"},
+	{"label": "Gilbert  Kipkorir Rono", "val": "2097-4"},
+	{"label": "Milka Cherono Samoei", "val": "3682-2"},
+	{"label": "Priscah Cherotich Kosgei", "val": "3876-0"},
+	{"label": "Hosea Kipkurgat Sitienei", "val": "2102-2"},
+	{"label": "Bridgid Jekorir Lagat", "val": "3877-8"},
+	{"label": "Jackline Cherono Kemei", "val": "3878-6"},
+	{"label": "Rebecca Jepkorir", "val": "3683-0"},
+	{"label": "Samuel Kimeli Bor", "val": "3684-8"},
+	{"label": "Eunice Chesang Kimoi", "val": "3685-5"},
+	{"label": "Everline Cheptoo Serem", "val": "3686-3"},
+	{"label": "LilianJepleting", "val": "2100-6"},
+	{"label": "Rael Jepkorir", "val": "2104-8"},
+	{"label": "Teresa Kurui", "val": "3687-1"},
+	{"label": "Eliud Koech", "val": "3879-4"},
+	{"label": "Barnabas Kimaru Koross", "val": "2101-4"},
+	{"label": "Edward Kiprop Norman", "val": "1824-2"},
+	{"label": "Peter Kipkemboi Birech", "val": "1825-9"},
+	{"label": "Peter Kibet Kemboi", "val": "1826-7"},
+	{"label": "Gladys Koros", "val": "3650-9"},
+	{"label": "Zipporah Chebet Boit", "val": "1828-3"},
+	{"label": "Mary Cherotich Barno", "val": "1829-1"},
+	{"label": "Nancy Jepchirchir", "val": "3651-7"},
+	{"label": "Edwin Kiplagat Letting", "val": "1831-7"},
+	{"label": "Ruth Cheptoo Kitur", "val": "1832-5"},
+	{"label": "Eunice Jerobon Melewa", "val": "1833-3"},
+	{"label": "DorcusJeptoo Ngelechei", "val": "3652-5"},
+	{"label": "Benjamin  Cheruiyot Yego", "val": "3653-3"},
+	{"label": "Musa Kipchumba Koech", "val": "1836-6"},
+	{"label": "Samuel Kiptanui Mzee", "val": "1837-4"},
+	{"label": "Sylvenas Kiprop Kurgat", "val": "3654-1"},
+	{"label": "Maximilah Jeptoo Chepkwony", "val": "1839-0"},
+	{"label": "Christine  Justine Yego", "val": "3655-8"},
+	{"label": "Dorcus Chepkemboi Murrey", "val": "1841-6"},
+	{"label": "Issac Kipngetich Timbwol", "val": "1842-4"},
+	{"label": "Selly Jepkurui Cheruiyot", "val": "3656-6"},
+	{"label": "Paul Kiptanui Maina", "val": "1845-7"},
+	{"label": "Caroline Chepkemboi Cheruiyot", "val": "1865-5"},
+	{"label": "HellenJepkemei Chirchir", "val": "1848-1"},
+	{"label": "Stanley  Kipkemboi Rugut", "val": "1850-7"},
+	{"label": "JosphineKemunto Murumbi", "val": "1851-5"},
+	{"label": "Mary Cheptoo Wendot", "val": "1863-0"},
+	{"label": "Martim Sarah Jelimo", "val": "3673-1"},
+	{"label": "Stanley  Kipleting Metto", "val": "1859-8"},
+	{"label": "Rose  Jepchumba Mugun", "val": "1862-2"},
+	{"label": "Lily Chepkemoi Matelong", "val": "1861-4"},
+	{"label": "Beatrice Ichenihi", "val": "3674-9"},
+	{"label": "Caroline Chepchumba", "val": "3675-6"},
+	{"label": "Jane Jeruto", "val": "1175-9"},
+	{"label": "Lilian Cherotich", "val": "2185-7"},
+	{"label": "Nelly Too", "val": "3676-4"},
+	{"label": "Julia Magoi", "val": "3677-2"},
+	{"label": "Paul Busienei", "val": "3678-0"},
+	{"label": "Silas Keronei", "val": "3679-8"},
+    ];
+
+    $('#encounter\\.provider_id_select').autocomplete({
```

Comment:
This probably should coordinate with @ShwethaThammaiah effort of pulling this logic to the main muzima.js.